predicate/predicate.py

```diff
@@ -11,9 +12,12 @@
 from django.db.models import Manager
 from django.db.models import QuerySet
 from django.db.models.constants import LOOKUP_SEP
+from django.db.models.query import REPR_OUTPUT_SIZE
 from django.db.models.query_utils import Q
 from django.utils.functional import cached_property
 
+from .constants import AND
+from .constants import OR
```

Comment:
What is this? `constants` module? I think it'd be preferable to use the constants defined on `Q` if needed (`Q.AND` / `Q.OR`).